### DISCUSSION TITLE
How do I change the table empty image?

### DISCUSSION THREAD

**Discussion by dudghks117** [ant-design-blazor/ant-design-blazor#1317] [Q&A]
When there is no data in Table, the default Empty image is Empty-Simple,

but I want to change it to Default-Empty image. What should I do?

**Comment by ElderJames** [comment]
Hi @dudghks117 ,thanks for contacting us.

Would you please help us to add a template for empty image? Just named `EmptyTemplate`, and put it in https://github.com/ant-design-blazor/ant-design-blazor/blob/4fc4afa26b18d88d8a79163436db055b2e5bcfe2/components/table/Table.razor#L155

**Comment by ElderJames** [comment]
We added this in #https://github.com/ant-design-blazor/ant-design-blazor/pull/3031
